Console/Release/Logger.h

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
I've already seen current file in this pull request. Please, remove copy of this file and use one.